tests/container

```diff
@@ -214,6 +214,7 @@ if ! { runsMinimumKernel 6.9 && hasNeededAPIExtension container_bpf_delegation;
         lxc init "${IMAGE}" c1
         lxc config set c1 security.delegate_bpf=true
         ! lxc start c1 || false
+        lxc config unset c1 security.delegate_bpf
```

Comment:
Ah, I had not look carefully at what baggage `c1` had accumulated. Considering the BPF delegation `else` branch, I think you were better off with a fresh instance.